templates/mockery.tmpl

```diff
@@ -0,0 +1,77 @@
+// Code generated by mockery; DO NOT EDIT.
```

Comment:
I wonder if mockery should add these code generation comments itself instead of templates providing it?

When it comes to the `template: mockio` comment, it might make more sense for mockery to inject this so that it becomes something like:

```go
// Code generated by mockery; DO NOT EDIT.
// github.com/vektra/mockery
// template: https://raw.githubusercontent.com/ovechkin-dm/mockio/bcd908070f77f7db95ec49ea9b16ff99ecac1cf2/templates/mockery.tmpl
```

This is just me musing to myself, happy to hear your thoughts.

Reply:
Maybe a parameter of some kind to overwrite it ?

Reply:
Both good ideas, however there are couple of caveats.
1. It could be the case that person don't want to mention the usage of mockery for codegen. So it is more up to you whether you want to force user to know or not.
2. Although template link could be helpful, I assume in most cases it will provide more confusion to user, because, for example, before start of our conversation I was not even aware of the template engine for mockery, and the link to template would not bring any useful information to me personally.